syncano/models/accounts.py

```diff
@@ -133,6 +137,18 @@ def reset_key(self):
         connection = self._get_connection()
         return connection.request('POST', endpoint)
 
+    def auth(self):
+        properties = self.get_endpoint_data()
+        endpoint = self._meta.resolve_endpoint('auth', properties)
+        connection = self._get_connection()
+
+        data = {
+            'username': connection.username,
+            'password' : connection.password
+        }
+
+        return connection.request('POST', endpoint, data=data)
```

Comment:
Yep, but there is other place to do that - this only simply to test credentials. For that, I think that we should refactor a little bit user auth in connection.py.